src/main/java/gregtech/common/covers/CoverMachineController.java

```diff
@@ -182,17 +183,30 @@ private IControllable getControllable() {
     private void resetCurrentControllable() {
         IControllable controllable = getControllable();
         if(controllable != null) {
-            controllable.setWorkingEnabled(true);
+            controllable.setWorkingEnabled(isOtherAllowingWork());
         }
     }
 
     private void updateRedstoneStatus() {
-        boolean shouldAllowWorking = getRedstoneSignalInput() < minRedstoneStrength;
-        if(isInverted) shouldAllowWorking = !shouldAllowWorking;
         IControllable controllable = getControllable();
         if(controllable != null) {
-            controllable.setWorkingEnabled(shouldAllowWorking);
+            controllable.setWorkingEnabled(shouldAllowWorking() && isOtherAllowingWork());
+        }
+    }
+
+    private boolean shouldAllowWorking(){
+        boolean shouldAllowWorking = getRedstoneSignalInput() < minRedstoneStrength;
```

Comment:
This does not work correctly if both `minRedstoneStrength` and `getRedstoneSignalInput()` are set to 15. But I am not sure how to handle this yet. I may debate this on Discord.

Reply:
if both are set to 15 then the controller don't work if we use `<=` instead of `<` then the machine will always work even when directly putting a lever on it.

Reply:
That would be correct statement but in this case it would lower requirements for Redstone signal by one level. Let's say from 8 to 7 which may break someone builds. And we always try to maintain high level of backward compatibility (not always possible). So in this case we need to add some special logic which will handle 15 case (and properly documented it - yes we have many hacks without documentation but I won't all any new ones missing documentation during my reign) and let's everything else work as before.

Reply:
as you can see here :
 https://github.com/GregTechCE/GregTech/blob/59cf924550f97635e1f576bf68309adcb8bface9/src/main/java/gregtech/common/covers/CoverMachineController.java#L190
my PR should not make any breaking change by reducing redstone requirement.
I understand that maintaining overall code quality is important don't worry I'll did any required changes until it become perfect.

Reply:
Yeah you are right I had that logic inverted in my head because this concept prevents working by default not allows it. My bad. This is correct implementation.